examples/dotnet-example02/appsettings.json

```diff
@@ -28,9 +28,9 @@
   },
   // Azure Content Safety settings
   "AzureContentSafety": {
-    "Endpoint": "https://....cognitiveservices.azure.com/",
+    "Endpoint": "https://openai1-x3boiusjqblbm.openai.azure.com/",
     "AuthType": "ApiKey",
-    "ApiKey": "..."
+    "ApiKey": "616c3edc5550418f801a27c90d6c01de"
```

Comment:
please undo these changes -- the key will remain in the repo history even after updating the PR, so I recommend invalidating it ASAP